core/controllers/dashboard_test.py

```diff
@@ -463,6 +479,58 @@ def test_no_explorations(self):
         self.assertEqual(response['explorations_list'], [])
         self.logout()
 
+    def test_no_explorations_and_visit_dashboard(self):
+        self.login(self.OWNER_EMAIL)
+        # Testing that creator only visit dashboard without any
+        # exploration created
```

Comment:
I think this can go on the previous line. Generally, try and fill as much as possible (upto the limit of upto 80 chars per line)